lab_1/main.py

```diff
@@ -11,26 +11,63 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
 
+    if isinstance(text, str):  # text является экземпляром классом str
+        text = text.lower()  # перевод строки в нижний регистр
+        for char in text:
+            if char != ' ' and not (char.isalpha()):  # проверка то,что символ не пробел и не является буквой
```

Comment:
good but if a line is `'a\ngirl'`?
you need to consider white space characters like `\n` or `\t` -> `isspace` check

Reply:
done